src/sidebar/Sidebar.js

```diff
@@ -14,19 +14,19 @@ import Template from "./Sidebar.blp" with { type: "uri" };
 
 import "../icons/edit-find-symbolic.svg";
 
-const GTK_INDEX = 19;
+let doc_index;
```

Comment:
Maybe to make it clear what it actually refers to?
```suggestion
let gtk_doc_index;
```

Reply:
I'll let this be for now